go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/golang/protobuf v1.5.3
 	github.com/hashicorp/go-version v1.6.0
+	github.com/huang-wei/kubernetes v1.14.9
```

Comment:
Why did remove `k8s.io/kubernetes v1.26.3` and add this one?

Reply:
Had some issues with my IDE, as discussed in the chat I will port in our code the necessary code.

Reply:
Ported everything in `apis/custom`